styleguide/source/_patterns/02-molecules/article-stub-as-inline.json

```diff
@@ -0,0 +1,13 @@
+{
+  "heading": {
+    "level": 4,
+    "text": "Example Label Text for an Inline Article Stub",
+    "class": "ama__article-stub-inline__label"
+  },
+  "link": {
+    "href": "http://www.google.com",
+    "text": "Example text for the article link, lorem ipsum dolor sit amet",
+    "target": "_self",
+    "class": "ama__article-stub-inline__link"
```

Comment:
There should be a double hyphen before `inline` in the class since it's a modifier